### PR TITLE
Remove unused files

### LINKED ISSUE #2073: Improve Resources/public directory
## Feature Request

There are a lot of images , one jQuery library (and I don't think it is ever used), and a css file.

We should see what is used, remove the rest and know what to do with the css file.

### PR DESCRIPTION
**Pull request by jordisala1991**
## Subject

I am targeting this branch, because on 3.x it could be considered a BC break

Closes #2073 .

## Changelog

```markdown
### Removed
- Removed unused doctrine skeleton files for SonataEasyExtends usage.
- Removed unused SonataAdminBundle translation files
- Removed unused jQuery Nivo Gallery dependency
```

### CODE REVIEW

**Review by VincentLanglet — COMMENTED** [review]
Are you sure about the translations ?

**Comment by jordisala1991** [comment]
I tried on sandbox. it is using SonataMediaBundle domain

**Comment by VincentLanglet** [comment]
Was added here: https://github.com/sonata-project/SonataMediaBundle/pull/1627
Don't know if something was fixed to correctly use the label_catalogue

**Review by VincentLanglet — APPROVED** [review]

**Comment by jordisala1991** [comment]
Not at all, not sure what error was fixed, since there is no issue linked, but it works without this files.

**Review by phansys — COMMENTED** [review]
Could you please provide other PR for the 3.x branch adding a `NEXT_MAJOR` comment for these removals?

**Comment by jordisala1991** [comment]
> Could you please provide other PR for the 3.x branch adding a `NEXT_MAJOR` comment for these removals?

Dont know what you mean. Mind to explain?

**Comment by VincentLanglet** [comment]
> > Could you please provide other PR for the 3.x branch adding a `NEXT_MAJOR` comment for these removals?
> 
> Dont know what you mean. Mind to explain?

Adding a "Next major remove this file" message in those files in a 3.x PR I think

File: src/Resources/translations/SonataAdminBundle.de.xliff [hunk 1]
```diff
@@ -1,14 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (core23) [inline src/Resources/translations/SonataAdminBundle.de.xliff:1]: This files should be kept. They are needed for this: https://github.com/sonata-project/SonataMediaBundle/blob/master/src/Resources/config/doctrine_orm_admin.php#L31

**Review by core23 — CHANGES_REQUESTED** [review]

**Review by core23 — APPROVED** [review]

**Comment by phansys** [comment]
>>>Could you please provide other PR for the 3.x branch adding a NEXT_MAJOR comment for these removals?
>>
>>Dont know what you mean. Mind to explain?
>
>Adding a "Next major remove this file" message in those files in a 3.x PR I think

That's exactly what I mean. Thank you.

**Review by phansys — APPROVED** [review]

**Pull request merged**